### ISSUE TITLE
DLQ key

### ISSUE THREAD

**Issue by ebrahimradi** [mateusjunges/laravel-kafka#173]
**Describe the bug**
I need to send a message to DLQ topic. Currently it is possible to send it to DLQ topic. 
The bug is key for the message that will go to DLQ topic. 
referring to  `\Junges\Kafka\Consumers\Consumer::sendToDlq` method,
there is a problem there:
I do not think the "key" is in "payload". the key is already in "$message".
Because end of `\Junges\Kafka\Consumers\Consumer::sendToDlq`, the key is null.

## Steps to reproduce

-  Create custom handler
```php
class MyHandler implements HandlerInterface
{
    public function __invoke(KafkaConsumerMessage $message): void
    {
        throw new \Exception('This class should be implemented.');
    }
}
```

- Create consumer with this handler and a DLQ topic
- Consumer should consume a message like this:
```json
{
  "headers": {
    "source": "mysource",
    "timestamp": "2023-02-09T10:19:26+01:00",
    "uuid": "a31e35d5-e2d4-4b95-9ca4-b5d83221582c"
  },
  "key": "234",
  "value": {
    "mykey":"myvalue"
  }
}
```
- So after exception i should see this message in DLQ topic but only value part is there.

**Comment by mateusjunges** [comment]
Hey @ebrahimradi thanks for reporting. I'll look into this as soon as I have some free time today :slightly_smiling_face:

**Comment by ebrahimradi** [comment]
I added this pull request if it can help :-) https://github.com/mateusjunges/laravel-kafka/pull/175

**Comment by mateusjunges** [comment]
@ebrahimradi released in v1.12.1, thanks!
